eledmac.dtx

```diff
@@ -10794,7 +10793,7 @@
 %    \end{macrocode}
 % \end{macro}
 % \subsubsection{Options for critical notes}
-% Before generating the options commands for critical notes, we check the \verb+nocritical+ option.
+% Before generating the optional commands for critical notes, we check the \verb+nocritical+ option.
```

Comment:
On Sun, Apr 12, 2015 at 02:58:02PM -0700, Ma�eul Rouquette wrote:
> > @@ -10794,7 +10793,7 @@
> >  %    \end{macrocode}
> >  % \end{macro}
> >  % \subsubsection{Options for critical notes}
> > -% Before generating the options commands for critical notes, we check the \verb+nocritical+ option.
> > +% Before generating the optional commands for critical notes, we check the \verb+nocritical+ option.
> 
> en gros les commandes d'option sont celles qui sont document�s au � 5.4 (de la version 1.20). Par exemple `\numberonlyfirstinline` `\lemmaseparator` etc.

Je vois. L'id�e est donc que ce type de commande permet de r�gler ou
d'ajuster l'apparence des notes critiques. Je mettrais alors: "Before
generating the commands that are used to set/adjust/tune the critical
notes". Les trois mots conviennent. Adjust est plut�t le r�glage qu'on
fait quand on tourne un bouton. Tune aussi (voir le tuner des radios).
Je prendrais "set".